dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -81,17 +81,11 @@
                 .labelblock
                   = t('signup_form.age')
                 = f.select :age, User::AGE_DROPDOWN_OPTIONS, include_blank: true, class: 'fieldblock'
-              - if ask_gender_experiment?(request, session.id.to_s)
-                - gender_input_type = gender_input_type?(request, session)
-                - SignUpTracking.log_gender_input_type_started(session, gender_input_type, request.locale, 'section_signup')
-                - if gender_input_type != User::GenderExperimentHelper::GENDER_INPUT_TYPE_NONE
-                  .itemblock
-                    .labelblock
-                      = t('signup_form.gender')
-                    - if gender_input_type == User::GenderExperimentHelper::GENDER_INPUT_TYPE_DROPDOWN
-                      = f.select :gender, gender_options
-                    - else
-                      = f.text_field :gender
+              - if experiment_value('gender')
+              .itemblock
+                .labelblock
+                  = t('signup_form.gender')
+                = f.text_field :gender_student_input
```

Comment:
I think the indentation is off? This isn't working, right?

Reply:
Correct, I fixed this in the latest commit